notebooks/125-torchvision-zoo-to-openvino/125-convnext-classification.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<p>where we can find info that this preprocessing should be used for your model?</p><p>maybe it is better to use transforms that provided with model weights enum?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
changed